src/Alert/Alert.js

```diff
@@ -0,0 +1,54 @@
+import ClassNames from 'classnames'
+import React, { PropTypes } from 'react'
+
+/**
+ * React <b>Alert</b> Component for Patternfly Web Components
```

Comment:
I don't have strict preference. I basically just took what you had in shims repo